src/ImpactX.cpp

```diff
@@ -252,6 +252,13 @@ namespace impactx
                 pp_element.get("g", g);
                 pp_element.get("K2", K2);
                 m_lattice.emplace_back( DipEdge(psi, rc, g, K2) );
+            } else if (element_type == "constf") {
```

Comment:
That looks good.

Do you want to add a minimal input set that uses (aka: "covers") this code path already in continuous integration (CI)? Does not need to be super meaningful yet, but would make sure the code does not segfault as we continue to add code to the repo.

Reply:
Just added a commit.  (See above.)